docs/overview/script-storage.md

```diff
@@ -0,0 +1,33 @@
+---
+title: 'Office Scripts file storage and ownership'
+description: 'Information about how Office Scripts are stored in OneDrive and transferred between owners.'
+ms.date: 11/13/2020
+localization_priority: Normal
+---
+
+# Office Scripts file storage and ownership
+
+Office Scripts are stored as **.osts** files in your OneDrive. This allows your scripts to exist outside any particular workbook. Your OneDrive settings control the shared access and permissions for these script **.osts** files; independent of any Excel settings.
+
+## File storage
+
+You Office Scripts are stored in your OneDrive. The **.osts** files are found in the **/Documents/Office Scripts/** folder. Any edits made these **.osts** files, such as renaming or deleting files, will be reflected in the Code Editor and Script Gallery.
+
+Scripts that are shared with one of your workbooks remain in the cloud and are not copied to any of your local or OneDrive folders. The **Make a Copy** button of the Code Editor saves a separate copy of the script in your OneDrive. Changes to the copy don't affect the original script.
+
+### Script Folders
+
+Adding folders to your OneDrive helps keep your scripts organized. Any folders under **/Documents/Office Scripts/** are displayed under the **My Scripts** section of the Code Editor. Please note that these folders cannot be created and deleted through the Code Editor. Likewise, scripts cannot be placed in to or out of folders through the Code Editor.
+
+![Some scripts contained in folders, as displayed in the Code Editor task pane](../images/script-folders.png)
+
+## File ownership and retention
+
+Office Scripts are stored on a user's OneDrive, they follow the same retention and deletion policies. To learn how to handle scripts that were created and shared by a user being removed from your organization, see [OneDrive retention and deletion](/onedrive/retention-and-deletion).
```

Comment:
```suggestion
Office Scripts are stored in a user's OneDrive. They follow the same retention and deletion policies. To learn how to handle scripts that were created and shared by a user being removed from your organization, see [OneDrive retention and deletion](/onedrive/retention-and-deletion).
```
Can you clarify which retention and deletion policies are being followed (second sentence)? "same" as which ones?